test/json_to_class.dart

```diff
@@ -40,7 +40,7 @@ class JsonToClass {
       ..writeln('@JsonSerializable(fieldRename: FieldRename.snake)')
       ..writeln('class ${object.name} {')
       ..writeAll(
-          object.keys.map((key) => '\t${key.value} ${_assertKey(key.key)};\n'))
+          object.keys!.map((key) => '\t${key.value} ${_assertKey(key.key)};\n'))
```

Comment:
This should mark all fields as nullable by default:
```suggestion
          object.keys!.map((key) => '\t${key.value}? ${_assertKey(key.key)};\n'))
```